pkgs/development/python-modules/termtosvg/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildPythonPackage, fetchFromGitHub, svgwrite, pyte, mock, unittest2, isPy3k }:
+
+buildPythonPackage rec {
+  pname = "termtosvg";
+  version = "0.3.0";
+
+  # tests broken on Python 2, standalone app that should be fine with Python 3
+  disable = !isPy3k;
```

Comment:
If you use `buildPythonApplication` above, you can use `python3` as input and get rid of this.